src/protocols/bdx/BdxMessages.h

```diff
@@ -309,6 +310,28 @@ struct DataBlock : public BdxMessage
 using Block    = DataBlock;
 using BlockEOF = DataBlock;
 
+/**
+ * A struct for representing BlockQueryWithSkip messages
+ */
+struct BlockQueryWithSkip : public BdxMessage
+{
+    /**
+     * @brief
+     *  Equality check method.
```

Comment:
nit: please delete this doc-comment: it does not  seem to help me read the code (it is obvious what operator== is) and takes up 4 vertical lines.

Reply:
Removed the unnecessary comment